app/src/main/java/io/gnosis/safe/utils/MnemonicAddressDerivation.kt

```diff
@@ -0,0 +1,53 @@
+package io.gnosis.safe.utils
+
+import okio.ByteString
+import pm.gnosis.crypto.HDNode
+import pm.gnosis.crypto.KeyGenerator
+import pm.gnosis.mnemonic.Bip39
+import pm.gnosis.model.Solidity
+import pm.gnosis.utils.asBigInteger
+import java.math.BigInteger
+
+interface MnemonicAddressDerivation {
```

Comment:
Derivation sounds like a verb and normally we use nouns for class-names

Reply:
Derivation might represend one single derivation. It would be more correct to call this class MnemonicAddressDerivator

Reply:
changed in 164da5b29bba2da94c71e9fe5ca461c071398f90